optaweb-employee-rostering-frontend/src/ui/header/Toolbar.tsx

```diff
@@ -117,48 +117,38 @@ export class ToolbarComponent extends React.Component<Props, ToolbarState> {
     );
     const { tenantList, currentTenantId } = this.props;
     const { isTenantSelectOpen } = this.state;
-    if (tenantList.size === 0) {
-      return (
-        <Toolbar
-          style={{
-            backgroundColor: '#0000', // transparent background for background image
-          }}
-        >
-          <ToolbarGroup />
-          {bellAndCog}
-        </Toolbar>
-      );
-    }
-
-    const currentTenant = tenantList.find(t => t.id === currentTenantId) as Tenant;
+    const currentTenantName = tenantList.find(t => t.id === currentTenantId)?.name ?? 'TENANT ERROR';
```

Comment:
Should this be i18n?

Reply:
You mean the `TENANT ERROR` string? I don't think so. I think it can only happen as a result of a bug - the component's props would have be a `currentTenantId` and a `tenantList` that doesn't have a tenant with such ID. This shouldn't be possible given enough test coverage. Therefore I consider the fallback string as a hypothetical situation. It will never hit the user's eyes so I think it's not worth translating.